domain/schema/model/sql/0019-unit.sql

```diff
@@ -119,3 +119,49 @@ CREATE TABLE unit_state_relation (
     FOREIGN KEY (unit_uuid)
     REFERENCES unit (uuid)
 );
+
+CREATE TABLE unit_agent_status (
+    unit_uuid TEXT NOT NULL PRIMARY KEY,
+    status_id INT NOT NULL,
+    message TEXT,
+    updated_at DATETIME,
+    CONSTRAINT fk_unit_agent_status_unit
+    FOREIGN KEY (unit_uuid)
+    REFERENCES unit (uuid),
```

Comment:
Should this not reference the unit agent table ?

Reply:
No. The unit_agent table is unrelated. It stores the version and hash etc specifically of the jujud agent binary. Think of it as akin to "unit_agent_jujud_binary_version_info".